semcore/link/src/style/link.shadow.css

```diff
@@ -52,7 +54,8 @@ SLink[keyboardFocused]::after {
 SLink[disabled] {
   opacity: var(--intergalactic-disabled-opacity, 0.3);
   cursor: default;
-  pointer-events: none; /* Disable link interactions */
+  pointer-events: none;
+  /* Disable link interactions */
```

Comment:
maybe stay on same line or before pointer-events?

Reply:
Sure, it's just auto-formatting, will fix it,